src/cmd-fetch

```diff
@@ -74,6 +76,10 @@ while true; do
         --force)
             FORCE_ARG=--force-nocache
             ;;
+        --autolock)
+            shift;
+            AUTOLOCK_VERSION=$1
+            ;;
```

Comment:
How to `--autolock=version` and `--strict` intersect?

Reply:
In practice, you likely can't use `--strict` if you use `--autolock`, but they're theoretically independent. Added a check to be nice and a comment.